custom/icds_reports/reports/take_home_ration.py

```diff
@@ -97,3 +82,115 @@ def _format_infrastructure_data(data):
                 filters
             ]
         ]
+
+    def get_consolidated_data(self, filters, order_by):
+        headers = ['State', 'District', 'Block', 'Sector', 'Awc Name', 'AWW Name', 'AWW Phone No.',
```

Comment:
`launched_column` is missing from headers

Reply:
That is intentional